### ISSUE TITLE
CFG cutoff

### ISSUE THREAD

**Issue by Bili-Sakura** [LALBJ/PAI#14]
I that you filter logits<0.1*max in CFG

```python
cutoff = torch.log(torch.tensor(0.1)) + scores.max(dim=-1, keepdim=True).values
```

I wonder is this vital or can I just use the CFG logits without filtering.
